fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -452,8 +452,7 @@ public TGetTaskInfoResult getTasks(TGetTasksParams params) throws TException {
             TTaskInfo info = new TTaskInfo();
             info.setTask_name(task.getName());
             info.setCreate_time(task.getCreateTime() / 1000);
-            // Now there are only MANUAL types of Tasks
-            info.setSchedule("MANUAL");
+            info.setSchedule(task.getType().name());
```

Comment:
better use Periodical （START xxxx-xx-xx EVERY (xxx xxxx)）format

Reply:
e.g. PERIODICAL (START 2022-06-30T11:09:30 EVERY(10 SECONDS))

Reply:
I think this is ok